fastlane/Fastfile

```diff
@@ -261,14 +254,15 @@ private_lane :no_binary do
 end
 
 desc "Print out the changelog since the last tagged release"
-private_lane :print_release_changelog do |options|
-  tool_name = options[:tool] || "fastlane"
-  old_version = options[:old_version]
+lane :show_changelog do |options|
+  old_version = options[:old_version] || current_version
 
-  changes = Actions.sh("git log --pretty='* %s' #{tool_name}/#{old_version}...HEAD --no-merges ..", log: $verbose).gsub("\n\n", "\n")
-  changes.gsub!("[#{tool_name}] ", "") # some commits are pre-fixed with the tool name in [ ]
+  changes = sh("git log --pretty='* %s' #{old_version}...HEAD --no-merges ..", log: $verbose).gsub("\n\n", "\n")
   changes.gsub!("[WIP]", "") # sometimes a [WIP] is merged
 
+  github_diff_url = "https://github.com/fastlane/fastlane/compare/#{old_version}...master"
+  sh "open #{github_diff_url}"
```

Comment:
Question: what happens if we execute `open` on a CI box (where this might be running on a shell independent from the graphical logged-in user session)?